patches/server/0977-Add-mob-spawner-ranges-changes-and-API.patch

```diff
@@ -0,0 +1,108 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: DavidTs93 <david.ts93@gmail.com>
+Date: Fri, 7 Apr 2023 19:28:43 +0300
+Subject: [PATCH] Add mob spawner ranges changes and API
+
+
+diff --git a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+index 8d442c5a498ecf288a0cc0c54889c6e2fda849ce..6938dc41816d1fe9cd4b850d060a46fed51602b5 100644
+--- a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
++++ b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+@@ -276,6 +276,14 @@ public class GlobalConfiguration extends ConfigurationPart {
+                 executor.setMaximumPoolSize(_chatExecutorMaxSize);
+             }
+         }
++
++        public MobSpawnerRange mobSpawnerRange;
++
++        public class MobSpawnerRange extends ConfigurationPart {
```

Comment:
Should this make some mention of the fact that it's the default range? So like DefaultMobSpawnerRange?

Otherwise it might be more likely to be misinterpreted as overriding any other changes plugins made.